python/kfserving/kfserving/test_storage.py

```diff
@@ -14,6 +14,16 @@
 
 import pytest
 import kfserving
+import os
+
+# Environment for testing
+GCS_PRIVATE_PATH = 'gs://bucket/path'
+GOOGLE_APPLICATION_CREDENTIALS = ''
+
+S3_PATH = 's3://bucket/path'
+S3_ENDPOINT = ''
+AWS_ACCESS_KEY_ID = ''
+AWS_SECRET_ACCESS_KEY = ''
 
```

Comment:
Is prepending with 'AWS_' required by Minio, even if its not AWS backend?

Reply:
These environment variable name are based on PR #94. If we want a different environment variable name, then we have to change the go controller as well.